sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-institution-designation-report.sql

```diff
@@ -0,0 +1,37 @@
+INSERT INTO
+  sims.report_configs (report_name, report_sql)
+VALUES
+  (
+    'Institution_Designation_Report',
+    'SELECT
+      institutions.operating_name AS "Institution Operating Name",
+      institution_locations.name AS "Location Name",
+      institution_locations.institution_code AS "Location Code",
+      institution_type.name AS "Institution Type",
+      designation_agreement_locations.requested AS "Request for designation",
+      designation_agreement_locations.approved AS "Approved for designation",
+      designation_agreements.designation_status AS "Designation Status",
+      designation_agreements.assessed_date AS "Assessed Date",
+      designation_agreements.end_date AS "Expiry Date",
+      institution_locations.primary_contact ->> ''firstName'' AS "Location Contact",
```

Comment:
Is `primary_contact` not supposed to be displayed with first and last names?
It may be added as two columns but it probably not be only the `firstName`.